### PR TITLE
Fix one click update test

### PR DESCRIPTION
**Pull request by diosmosis**
@tsteur this was a test issue it seems, not whatever you were experiencing on demo or cloud

### CODE REVIEW

**Comment by tsteur** [comment]
@diosmosis I suppose this PR is to fix https://github.com/matomo-org/matomo/issues/16430 ?

File: tests/PHPUnit/Fixtures/LatestStableInstall/GitCommitReleaseChannel.php [hunk 1]
```diff
@@ -20,7 +21,10 @@ public function getName()
 
     public function getUrlToCheckForLatestAvailableVersion()
     {
-        return 'http://' . Url::getHost(false) . '/tests/resources/one-click-update-version.php';
+        $majorVersion = (int) Version::VERSION;
+        $majorVersion += 1;
```
Comment (tsteur) [inline tests/PHPUnit/Fixtures/LatestStableInstall/GitCommitReleaseChannel.php:25]: btw once we are on Matomo 4 will we need to remove this one again?

Comment (tsteur) [reply tests/PHPUnit/Fixtures/LatestStableInstall/GitCommitReleaseChannel.php:25]: I see it should not be needed

**Comment by tsteur** [comment]
@diosmosis the tests seem to fail now. Not sure if it's related to this? see eg https://travis-ci.org/github/matomo-org/matomo/jobs/727808766#L641

**Comment by diosmosis** [comment]
@tsteur That means nvm (the node version manager) isn't on that machine... might be a random failure, but related to travis-ci

**Comment by diosmosis** [comment]
> @diosmosis I suppose this PR is to fix #16430 ?

it's related, but it's a test issue only. whatever you experienced when upgrading to matomo 4 isn't reproduced by our oneclickupdate test.

**Comment by tsteur** [comment]
@diosmosis seems some tests are still failing otherwise seems good

**Pull request merged**